diff --git a/www/controllers/MeetingController.php b/www/controllers/MeetingController.php
index 9c92ce0..2b465c3 100644
--- a/www/controllers/MeetingController.php
+++ b/www/controllers/MeetingController.php
@@ -955,8 +955,121 @@ class MeetingController {
 #    }
 #  }
 
+  static public function meetingDetails_new ($category,$meetid,$itemid) {
+
+    $m = getDatabase()->one(" 
+      select * 
+			from meeting 
+			where meetid = :meetid ",
+			array("meetid" => $meetid)
+		);
+    if (!$m['id']) {
+      self::doList($category);
+      return;
+    }
+
+    $title = meeting_category_to_title($m['category']);
+    $focusFrameSrc = self::getDocumentUrl($meetid,'AGENDA');
+		$focusFrameSrcBase = self::getDocumentUrl($meetid,'AGENDA');
+
+    $items = getDatabase()->all(" select * from item where meetingid = :meetingid order by id ",array("meetingid"=>$m['id']));
+
+    top3($title . " on " . substr($m['starttime'],0,10));
+
+		?>
+		<div class="row">
+		<?php
+		# generate item and file list
+		{
+		?>
+		<div class="col-sm-4">
+    <div id="agendanav" style="overflow:scroll; height: 620px;">
+
+		<h3 class="text-center"><?php print $title; ?></h3>
+
+		<div class="row" style="margin-bottom: 10px;">
+		<div class="col-sm-6 text-center"><?php print "".substr($m['starttime'],0,10); ?></div>
+		<div class="col-sm-6 text-center"><?php print "<a href=\"http://app05.ottawa.ca/sirepub/mtgviewer.aspx?meetid=".$meetid."&doctype=AGENDA\"> view on Ottawa.ca <small><i class=\"fa fa-external-link\"></i></small></a>"; ?></div>
+		</div>
+
+    <table id="itemTable" class="table table-bordered table-hover table-condensed" style="width: 95%;">
+		<tr>
+			<th>Item/File</th>
+			<th>Ext.</th>
+		</tr>
+		<?php
+		foreach ($items as $i) {
+			?>
+			<tr>
+			<td style="text-transform: capitalize;">
+      <?php print "<a href=\"javascript:focusOnItem({$i['itemid']})\">".strtolower($i['title'])."</a>"; ?>
+			</td>
+			<td><?php print " <a target=\"_blank\" href=\"http://app05.ottawa.ca/sirepub/item.aspx?itemid={$i['itemid']}\"><small><i class=\"fa fa-external-link \"></i></small></a>"; ?></td>
+			</tr>
+			<?php
+
+      $files = getDatabase()->all(" select * from ifile where itemid = :itemid order by id ",array("itemid"=>$i['id']));
+      if (count($files) > 0) {
+        foreach ($files as $f) {
+          $ft = self::trimFileTitle($i['title'],$f['title']);
+          $fileurl = OttWatchConfig::WWW . "/meetings/file/" . $f['fileid'];
+          print "
+					<tr>
+					<td style=\"padding-left: 20px;\">
+					<i class=\"fa fa-file-text\"></i>
+					{$ft}
+					</td>
+					<td>
+					<a target=\"_blank\" href=\"{$fileurl}\"><i class=\"fa fa-external-link\"></i></a>
+					</td>
+					</tr>
+					";
+        }
+      }
+			?>
+			<?php
+		}
+		?>
+		</table>
+		</div><!-- .scroll -->
+		</div>
+		<?php } ?><!-- .generate nav -->
+
+		<div class="col-sm-8">
+			<h3 id="agendaDiv" class="text-center">Agenda</h3>
+      <p class="hidden-sm hidden-md hidden-lg text-center"><a href="javascript:scrollToItems()">Back to Item Display</a></p>
+	    <iframe id="focusFrame" src="<?php print $focusFrameSrc; ?>" style="width: 100%; height: 600px; border: 2px solid #c0c0c0;"></iframe>
+		</div>
+
+		</div><!-- .row -->
+
+		<script>
+    function scrollToItems() {
+	    $('html, body').animate({
+				scrollTop: $("#itemTable").offset().top
+			}, 100);
+		}
+    function focusOnItem(id,title) {
+      $('#focusFrame').attr('src','<?php print $focusFrameSrcBase; ?>#Item' + id);
+	    $('html, body').animate({
+				scrollTop: $("#agendaDiv").offset().top
+			}, 100);
+      return;
+		}
+		</script>
+
+		<?php
+
+		bottom3();
+	}
+
   static public function meetingDetails ($category,$meetid,$itemid) {
 
+		if ($_GET['new'] == 1) {
+			self::meetingDetails_new($category,$meetid,$itemid);
+			return;
+		}
+
     $m = getDatabase()->one(" 
       select 
         *,
diff --git a/www/index.php b/www/index.php
index 2c59a8a..350cc77 100644
--- a/www/index.php
+++ b/www/index.php
@@ -255,28 +255,6 @@ function dashboard() {
   part of the political conversation in Ottawa. 
   <a href="about">Read about all the features</a>.
 
-	<!--
-  <div style="background: #08c; color: #ffffff; padding: 20px; font-size: 200%; border-radius: 4px;">
-  <center>
-  <a href="/election/" style="color: #ffffff;">
-  <i class="fa fa-check-square-o fa-4" style="font-size: 125%;"></i>
-  Election Coverage 
-  <i style="font-size: 75%"><b>#ottvote</b></i>
-  </center>
-  </a>
-  </div>
-	-->
-	<!--
-  <div style="margin-top: 10px; background: #08c; color: #ffffff; padding: 20px; font-size: 200%; border-radius: 4px;">
-  <center>
-  <a href="/election/listDonations" style="color: #ffffff;">
-  <i class="fa fa-search fa-4" style="font-size: 125%;"></i>
-	Campaign Donations
-  </center>
-  </a>
-  </div>
-	-->
-
   <div style="margin-top: 10px; background: #08c; color: #ffffff; padding: 20px; font-size: 125%; border-radius: 4px;">
   <center>
 	<?php
@@ -509,7 +487,17 @@ function error404() {
   bottom();
 }
 
+function top3($title = '',$quiet = false) {
+	top_common(1,$title,$quiet);
+	?>
+	<?php
+}
+
 function top($title = '',$quiet = false) {
+	top_common(0,$title,$quiet);
+}
+
+function top_common($v3, $title = '',$quiet = false) {
   global $OTT_WWW;
 ?>
 <!DOCTYPE html>
@@ -519,12 +507,56 @@ function top($title = '',$quiet = false) {
 <meta property="twitter:account_id" content="1512911885" /><!-- ads.twitter -->
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="<?php print $OTT_WWW; ?>/bootstrap/css/bootstrap.min.css" rel="stylesheet" media="screen" type="text/css">
-<link href="<?php print $OTT_WWW; ?>/bootstrap/css/bootstrap-responsive.css" rel="stylesheet">
-<link href="//netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.css" rel="stylesheet">
+<?php 
+if ($v3) {
+	?>
+	<link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
+	<link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.0.3/css/bootstrap.min.css">
+	<link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.0.3/css/bootstrap-theme.min.css">
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+	<script src="//netdna.bootstrapcdn.com/bootstrap/3.0.3/js/bootstrap.min.js"></script>
+	<?php
+} else {
+	?>
+	<link href="//netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.css" rel="stylesheet">
+	<link href="<?php print $OTT_WWW; ?>/bootstrap/css/bootstrap.min.css" rel="stylesheet" media="screen" type="text/css">
+	<link href="<?php print $OTT_WWW; ?>/bootstrap/css/bootstrap-responsive.css" rel="stylesheet">
+	<script src="<?php print $OTT_WWW; ?>/jquery.js" type="text/javascript"></script>
+	<script src="<?php print $OTT_WWW; ?>/bootstrap/js/bootstrap.min.js"></script>
+	<?php
+}
+?>
 <style type="text/css">
+	<?php 
+	if ($v3) {
+		?>
+	html { 
+		position: relative; 
+		min-height: 100%; 
+	}
   body {
-	  padding: 20px;
+		margin-bottom: 60px;
+	}
+		<?php
+	} else {
+		?>
+  body {
+		padding: 20px;
+	}
+		<?php
+	}
+	?>
+	.footer {
+	  position: absolute;
+	  bottom: 0;
+	  width: 100%;
+	  /* Set the fixed height of the footer here */
+	  height: 60px;
+	  background-color: #f5f5f5;
+	}
+	.container {
+	  width: auto;
+	  padding: 0 15px;
 	}
 	.ottwatchstorybody {
 		font-size: 110%;
@@ -550,8 +582,6 @@ function top($title = '',$quiet = false) {
 		color: #ffffff;
 	}
 </style>
-<script src="<?php print $OTT_WWW; ?>/jquery.js" type="text/javascript"></script>
-<script src="<?php print $OTT_WWW; ?>/bootstrap/js/bootstrap.min.js"></script>
 <script>
 function copyToClipboard (text) {
   window.prompt ("Copy to clipboard: Ctrl+C, Enter", text);
@@ -586,149 +616,184 @@ if ($quiet) { return; }
 $remaining = getDatabase()->one(" select count(1) c from candidate_donation where amount is null ");
 $remaining = $remaining['c'];
 #$remaining = 0;
-?>
 
-<div class="row-fluid">
-<div class="span12">
-<div class="navbar">
-<div class="navbar-inner">
-<ul class="nav">
-<li><a href="<?php print $OTT_WWW; ?>">Home</a></li>
-<?php
-if (!LoginController::isLoggedIn()) {
-  ?>
-  <li><a href="<?php print $OTT_WWW; ?>/user/login?next=<?php print urlencode($_SERVER['REQUEST_URI']); ?>">Login</a></li>
-  <?php
+if ($v3) {
+	?>
+
+	<nav class="navbar navbar-default" role="navigation">
+  <div class="container-fluid">
+    <!-- Brand and toggle get grouped for better mobile display -->
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a class="navbar-brand" href="/">OttWatch</a>
+    </div>
+
+    <!-- Collect the nav links, forms, and other content for toggling -->
+    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+      <ul class="nav navbar-nav">
+				<?php
+				if (!LoginController::isLoggedIn()) {
+				  ?>
+				  <li><a href="<?php print $OTT_WWW; ?>/user/login?next=<?php print urlencode($_SERVER['REQUEST_URI']); ?>">Login</a></li>
+				  <?php
+				} else {
+				  ?>
+				  <li><a href="<?php print $OTT_WWW; ?>/user/home">Profile</a></li>
+				  <li><a href="<?php print $OTT_WWW; ?>/user/logout?next=<?php print $_SERVER['REQUEST_URI']; ?>">Logout</a></li>
+				  <?php
+				}
+				?>
+				<li><a href="<?php print $OTT_WWW; ?>/about">About</a></li>
+				<li>
+					<?php
+					{
+				  $row = getDatabase()->one(" select * from story where deleted = 0 and published = 1 order by updated desc limit 1 ");
+					print "<a href=\"/story/{$row['id']}\">";
+					print "<b>Latest Story: {$row['title']}</b>";
+					print "</a>";
+					}
+					?>
+				</li>
+				<?php
+				if ($remaining == 0) {
+					?>
+					<!--<li><a style="" href="<?php print $OTT_WWW; ?>/election/processDonation/">0 donations remaining</a></li>-->
+					<?php
+				} else {
+					?>
+					<li><a style="font-weight: bold; color: #f00;" href="<?php print $OTT_WWW; ?>/election/processDonation/">HELP: <?php print $remaining; ?> donations left to scan!</a></li>
+					<?php
+				}
+				?>
+			</ul>
+		</div>
+	</div>
+	</nav>
+	<div class="container">
+
+	<?php
 } else {
-  ?>
-  <li><a href="<?php print $OTT_WWW; ?>/user/home">Profile</a></li>
-  <li><a href="<?php print $OTT_WWW; ?>/user/logout?next=<?php print $_SERVER['REQUEST_URI']; ?>">Logout</a></li>
-  <?php
-}
-?>
-<li><a href="<?php print $OTT_WWW; ?>/about">About</a></li>
-<li>
+	?>
+	<div class="row-fluid">
+	<div class="span12">
+	<div class="navbar">
+	<div class="navbar-inner">
+	<ul class="nav">
+	<li><a href="<?php print $OTT_WWW; ?>">Home</a></li>
 	<?php
-	{
-  $row = getDatabase()->one(" select * from story where deleted = 0 and published = 1 order by updated desc limit 1 ");
-	print "<a href=\"/story/{$row['id']}\">";
-	print "<b>Latest Story: {$row['title']}</b>";
-	print "</a>";
+	if (!LoginController::isLoggedIn()) {
+	  ?>
+	  <li><a href="<?php print $OTT_WWW; ?>/user/login?next=<?php print urlencode($_SERVER['REQUEST_URI']); ?>">Login</a></li>
+	  <?php
+	} else {
+	  ?>
+	  <li><a href="<?php print $OTT_WWW; ?>/user/home">Profile</a></li>
+	  <li><a href="<?php print $OTT_WWW; ?>/user/logout?next=<?php print $_SERVER['REQUEST_URI']; ?>">Logout</a></li>
+	  <?php
 	}
 	?>
-</li>
-<?php
-if ($remaining == 0) {
-	?>
-	<!--<li><a style="" href="<?php print $OTT_WWW; ?>/election/processDonation/">0 donations remaining</a></li>-->
+	<li><a href="<?php print $OTT_WWW; ?>/about">About</a></li>
+	<li>
+		<?php
+		{
+	  $row = getDatabase()->one(" select * from story where deleted = 0 and published = 1 order by updated desc limit 1 ");
+		print "<a href=\"/story/{$row['id']}\">";
+		print "<b>Latest Story: {$row['title']}</b>";
+		print "</a>";
+		}
+		?>
+	</li>
 	<?php
-} else {
+	if ($remaining == 0) {
+		?>
+		<!--<li><a style="" href="<?php print $OTT_WWW; ?>/election/processDonation/">0 donations remaining</a></li>-->
+		<?php
+	} else {
+		?>
+		<li><a style="font-weight: bold; color: #f00;" href="<?php print $OTT_WWW; ?>/election/processDonation/">HELP: <?php print $remaining; ?> donations left to scan!</a></li>
+		<?php
+	}
 	?>
-	<li><a style="font-weight: bold; color: #f00;" href="<?php print $OTT_WWW; ?>/election/processDonation/">HELP: <?php print $remaining; ?> donations left to scan!</a></li>
+	</ul>
+	</div>
+	</div>
+	</div>
+	</div>
 	<?php
+	}
+
 }
-?>
-</ul>
-</div>
-</div>
-</div>
-</div>
 
-<?php if (false) { ?>
+function bottom3($quiet = false) {
+	bottom_common(1,$quiet);
+}
 
-<div class="top-action-alert">
-<a href="/story/21/in-support-of-ottawa123-and-ranked-ballots">Action: In Support of Ottawa123 and Ranked Ballots <i class="fa fa-share"></i></a>
-</div>
+function bottom($quiet = false) {
+	bottom_common(0,$quiet);
+}
 
-<?php } ?>
+function bottom_common($v3, $quiet) {
+  global $OTT_WWW;
 
-<?php
-	if ($title != '') {
-    if (0) {
+  if ($v3) {
 		?>
-		<div style="background: #fcfcfc; padding: 10px; border: #c0c0c0 solid 1px;">
-		<div class="row-fluid">
-		<div class="lead span6">
-		<?php print $title; ?>
-		</div>
-		</div>
-		</div>
+		</div><!-- .container -->
 		<?php
-    }
 	}
-}
-
-function bottom($quiet = false) {
-  global $OTT_WWW;
 
-  if (!$quiet) {
-  ?>
-
-<div class="well" style="margin-top: 10px;" >
-
-<a href="<?php print $OTT_WWW; ?>"><img style="float: right; padding-left: 5px; width: 50px; height: 50px;" src="<?php print $OTT_WWW; ?>/img/ottwatch.png"/></a>
-<i>Created by <a href="http://kevino.ca"><b>Kevin O'Donnell</b></a> to make it easier to be part of the political conversation in Ottawa.</i><br/>
-On Twitter? Follow <b><a href="http://twitter.com/OttWatch">@OttWatch</a></b> and <b><a href="http://twitter.com/ODonnell_K">@ODonnell_K</a></b>
-
-<!--
-<div class="row-fluid">
-<div class="span4">
-</div>
-<div class="span2">
-</div>
-<div class="span2" style="text-align: right;">
-<div style="float: right;">
-<div class="fb-like" data-href="https://www.facebook.com/pages/Ottwatch/255793871239922" data-width="The pixel width of the plugin" data-height="50" data-colorscheme="light" data-layout="box_count" data-action="like" data-show-faces="true" data-send="false"></div>
-</div>
-On Facebook?&nbsp;
-</div>
-<div class="span4">
-</div>
-</div>
--->
-
-<!--
-<div id="clock">
-<script language="JavaScript">
-TargetDate = "10/27/2014 6:00 AM";
-BackColor = "ffffff";
-ForeColor = "ed1b24";
-CountActive = true;
-CountStepper = -1;
-LeadingZero = true;
-DisplayFormat = "<span class=\"clockdigit\">%%D%%</span> days until election day!";
-FinishMessage = "It is finally here!";
-</script>
-<script language="JavaScript" src="<?php print $OTT_WWW; ?>/countdown.js"></script>
-</div>
--->
-
-
-<div class="clearfix"></div>
-</div>
+  if (!$quiet && $v3) {
+		?>
+		<footer class="footer" style="padding: 10px;">
+		<div class="row">
+			<div class="col-sm-3 text-center">
+				<b>Be part of the political conversation in Ottawa.</b>
+			</div>
+			<div class="col-sm-3 text-center">
+				Created by <a href="http://kevino.ca"><b>Kevin O'Donnell</b></a>
+			</div>
+			<div class="col-sm-3 text-center">
+				Follow <a href="http://twitter.com/OttWatch">@OttWatch</a></b> and <b><a href="http://twitter.com/ODonnell_K">@ODonnell_K</a>
+			</div>
+			<div class="col-sm-3 text-center">
+				<a href="<?php print $OTT_WWW; ?>"><img style="width: 50px; height: 50px;" src="<?php print $OTT_WWW; ?>/img/ottwatch.png"/></a>
+			</div>
+		</div>
+		</footer>
+		<?php
+	}
+  if (!$quiet && !$v3) {
+	  ?>
+		<div class="well" style="margin-top: 10px;" >
+		<a href="<?php print $OTT_WWW; ?>"><img style="float: right; padding-left: 5px; width: 50px; height: 50px;" src="<?php print $OTT_WWW; ?>/img/ottwatch.png"/></a>
+		<i>Created by <a href="http://kevino.ca"><b>Kevin O'Donnell</b></a> to make it easier to be part of the political conversation in Ottawa.</i><br/>
+		On Twitter? Follow <b><a href="http://twitter.com/OttWatch">@OttWatch</a></b> and <b><a href="http://twitter.com/ODonnell_K">@ODonnell_K</a></b>
+		<div class="clearfix"></div>
+		</div>
 
-  <?php
+	  <?php
   }
-  googleAnalytics();
-  ?>
-
-    <script type="text/javascript">
-    /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
-    var disqus_shortname = 'ottwatch'; // required: replace example with your forum shortname
-
-    /* * * DON'T EDIT BELOW THIS LINE * * */
-    (function () {
-        var s = document.createElement('script'); s.async = true;
-        s.type = 'text/javascript';
-        s.src = 'http://' + disqus_shortname + '.disqus.com/count.js';
-        (document.getElementsByTagName('HEAD')[0] || document.getElementsByTagName('BODY')[0]).appendChild(s);
-    }());
 
+  googleAnalytics();
 
-    </script>
+  ?>
 
-  </body>
-  </html>
+	<script type="text/javascript">
+	var disqus_shortname = 'ottwatch'; // required: replace example with your forum shortname
+	(function () {
+		var s = document.createElement('script'); s.async = true;
+		s.type = 'text/javascript';
+		s.src = 'http://' + disqus_shortname + '.disqus.com/count.js';
+		(document.getElementsByTagName('HEAD')[0] || document.getElementsByTagName('BODY')[0]).appendChild(s);
+	}());
+	</script>
+	
+	</body>
+	</html>
   <?php
 }
 
@@ -790,3 +855,6 @@ function mapToTable($o) {
 
 ?>
 
+
+
+
